README.md

```diff
@@ -140,18 +141,38 @@ data = {"image": rgb, "K": intrinsics}
 predictions = model(data, {})
 ```
 
-Please visit [Hugging Face](https://huggingface.co/lpiccinelli) to access the repo models with weights. You can load V1 of UniDepth as:
+Please visit [Hugging Face](https://huggingface.co/lpiccinelli) to access the repo models with weights. You can load UniDepth as:
 
 ```python
-from unidepth.models import UniDepthV1
+from unidepth.models import UniDepthV1, UniDepthV2
 
 model = UniDepthV1.from_pretrained(f"lpiccinelli/unidepth-v1-{backbone}")
+# if you want to load V2 from HuggingFace:
+# model = UniDepthV2.from_pretrained(f"lpiccinelli/unidepth-v2-{backbone}")
```

Comment:
I guess we don't need comments here, right?

Just model_v1 = ... and model_v2 = ...